easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1.eb

```diff
@@ -16,7 +18,7 @@ large_src = 'large.tgz'
 timing_src = 'timing.tgz'
 sources = [
     'v%(version)s.tar.gz',
-    lapack_src,
+    (lapack_src, 'mv %s/lapack-netlib %s/lapack-netlib.old; mkdir %s/lapack-netlib; tar -C %s/lapack-netlib --strip-components=1 -zxf %%s' % (obv, obv, obv, obv)),
```

Comment:
@akesandgren I would try and shorten this a bit (lines should be < 120 chars long); also, I see no point in keeping `lapack-netlib.old` around...

```
lapack_unpack_cmd = "tar -C lapack-netlib --strip-components=1 -zxf %%s"
sources = [
    'v%(version)s.tar.gz',
    (lapack_src, "cd %%(name)s-%%(version)s; rm -rf lapack-netlib/*; %s; cd -" % lapack_unpack_cmd
    ...
```